src/TestFramework/TestFramework.Extensions/Attributes/DeploymentItemAttribute.cs

```diff
@@ -6,7 +6,8 @@
 namespace Microsoft.VisualStudio.TestTools.UnitTesting;
 
 /// <summary>
-/// Used to specify deployment item (file or directory) for per-test deployment.
+/// Used to specify deployment item (file or directory) for copying files or folders specified as deployment items to the deployment directory.
+/// Deployment directory is where all the deployment items are present along with TestSource dll.
```

Comment:
We should probably add a link to the corresponding property of the testcontext object.